app/src/main/java/nl/tudelft/bbw/controller/CryptoController.java

```diff
@@ -58,4 +64,37 @@ public final String decryptString(String data) {
             throw new RuntimeException("Error - Could not decode string: " + e);
         }
     }
+
+    /**
+     * encryptFile method
```

Comment:
I can see them now, and would like to refactor that at well. That code was not properly reviewed by me; I see no reason to repeat the method name appended with 'method' in the first line of the javadoc. Consistency is thus not important.